solr/core/src/java/org/apache/solr/cluster/placement/impl/PlacementContextImpl.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.solr.cluster.placement.impl;
+
+import org.apache.solr.client.solrj.cloud.SolrCloudManager;
+import org.apache.solr.cluster.Cluster;
+import org.apache.solr.cluster.placement.AttributeFetcher;
+import org.apache.solr.cluster.placement.PlacementContext;
+import org.apache.solr.cluster.placement.PlacementPlanFactory;
+
+import java.io.IOException;
+
+/**
+ *
+ */
+public class PlacementContextImpl implements PlacementContext {
```

Comment:
Maybe have "Simple" somewhere in the name of this class given it's instantiating `SimpleClusterAbstractionsImpl`?